### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by tcoupland**

### CODE REVIEW

File: test/onyx/integration/s3_test.clj [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  {:zookeeper/address "localhost:2181"
+   :onyx/tenancy-id "1"
+   :onyx.peer/job-scheduler :onyx.job-scheduler/balanced
+   :onyx.messaging/impl :aeron
+   :onyx.messaging/peer-port 40200
+   :onyx.messaging/bind-addr "localhost"})
+
+(def creds {:access-key "ADD_YOUR_OWN" ;ditto for below
+            :secret-key "ADD_YOUR_OWN"
+            :endpoint "eu-west-1"})
```
Comment (acron0) [inline test/onyx/integration/s3_test.clj:39]: Are these going to be env'd or removed?

Comment (tcoupland) [inline test/onyx/integration/s3_test.clj:39]: Yeah, they are just there so it can be made to work locally

Comment (acron0) [inline test/onyx/integration/s3_test.clj:39]: I think it's a bad idea to invite people to add their AWS creds. Can we either use environ or just let amazonica pick them up and advise people via the README how to get their creds in?

Comment (tcoupland) [inline test/onyx/integration/s3_test.clj:39]: they won't be mentioned at all once this becomes deployable. This is literally just a tide me over until this becomes deployable.

Comment (acron0) [inline test/onyx/integration/s3_test.clj:39]: As long as you pinky promise me :shipit:

**Pull request merged**